pkg/storage/tsdb/util.go

```diff
@@ -20,3 +21,13 @@ func HashBlockID(id ulid.ULID) uint32 {
 	}
 	return h
 }
+
+// ChunkRefEncode encodes a segment file number and byte offset into a ChunkRef.
+func ChunkRefEncode(segment uint32, offset uint32) chunks.ChunkRef {
+	return chunks.ChunkRef(uint64(segment)<<32 | uint64(offset))
+}
+
+// ChunkRefDecode decodes a ChunkRef into a segment file number and byte offset.
+func ChunkRefDecode(ref chunks.ChunkRef) (uint32, uint32) {
```

Comment:
This is duplicating code from `chunks` package (`chunks.NewBlockChunkRef` and `chunks.BlockChunkRef.Unpack`)

Reply:
thanks for pointing this out, I've removed these functions and their associated unit tests.